### ISSUE TITLE
Cleanup cookies and localstorage for a single container.

### ISSUE THREAD

**Issue by Lusito** [Lusito/forget-me-not#23]
Probably within the "cleanup" tab, an option to clean cookies only for the active container.

**Comment by BreadYogurt** [comment]
I would appreciate the option to have cookie rules apply to individual containers, like what Cookie Autodelete has.

**Comment by Lusito** [comment]
That should be doable. Could you provide some User stories (i.e. use-cases) which describe how you want to be able to use it? Thanks!

**Comment by BreadYogurt** [comment]
The way Cookie AutoDelete does it, each container has its own set of rules. "No Container" is also treated as a container in this way, getting its own set of rules. This works greatly for me, since I use containers with the intention of browsing certain sites with them. I don't want all the cookie rules I use for normal browsing, and often I don't want cookies from sites I visit in containers stored for my main browsing session. Basically, since containers keep their own cookies, I find it best that they also keep their own cookie rules.

**Comment by Lusito** [comment]
Okay, so I gather there should be these possible choices when selecting what a rule applies to:
- One entry for each container that currently exists
  - This includes one entry for the default container.. I guess this is what you refer to as "No Container"
- One entry that applies to all containers, so you can block cookies of, say facebook or google, without having to define that rule for every container you have.

**Comment by BreadYogurt** [comment]
Yep, that first point and its subpoint is the functionality I desire. The second point also sounds useful.

**Comment by pcause** [comment]
bump.  this would be a great addition.  i use the extension on android but need containers for windows/linux
